docs/developer/releases.rst

```diff
@@ -219,31 +218,31 @@ To build the beta, follow these steps:
      # The full version, including alpha/beta/rc tags.
     -release = '3.13.0'
     +release = '3.13.0b0'
-     
+
      # The language for content autogenerated by Sphinx. Refer to documentation
      # for a list of supported languages.
     diff --git a/setup.py b/setup.py
     index 44566ff5..74297bb5 100644
     --- a/setup.py
     +++ b/setup.py
     @@ -42,7 +42,7 @@ def get_requirements(requirements_file='requirements.txt'):
-     
+
```

Comment:
There are a lot of unrelated style changes in this file as well. Actually I do like removing unneeded whitespace, but let's do it in a separate pull request.

Reply:
Yeah sorry about that it wasn't intentional.